webpack.config.js

```diff
@@ -35,7 +35,8 @@ const config = module.exports = {
     ]
   },
   entry: {
-    'metamaps.bundle': './frontend/src/index.js'
+    'metamaps.bundle': './frontend/src/index.js',
+    'homepage': './frontend/src/homepage.js',
```

Comment:
if i had my way, you'd just move this into one bundle. 

if you want to make the homepage load  faster NOW, then leave it as is. But before merge can we use `npm ls` or something to figure out if any dependencies overlap? Well-designed modules won't overlap because webpack has module scope, but something like React will break if it's loaded twice (that's what's burned me before).

Code-splitting: I also don't understand it, but it's on my roadmap. Maybe I'll learn it at my new job and can contribute back